modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -1483,11 +1502,14 @@ private MockLiferayPortletRenderRequest _getMockLiferayPortletRenderRequest(
 		mockLiferayPortletRenderRequest.setAttribute(
 			WebKeys.COMPANY_ID, _company.getCompanyId());
 
+		MockLiferayPortletURL mockLiferayPortletURL =
```

Comment:
You change the code of the test, adding the value mockLiferayPortletURL.toString() to the ThemeDisplay, but this is never used in the test, so it is kind of confusing, to init values that you never use, but are explicit in the tests.

Reply:
true, I'll be more explicit next time ;)